BALSAMIC/commands/config/case.py

```diff
@@ -117,6 +119,7 @@ def case_config(
     panel_bed: Path,
     pon_cnn: Path,
     quality_trim: bool,
+    sentieon_install_dir: Path,
```

Comment:
Perhaps it makes sense to include both to standardise the implementation. What do you think?

```suggestion
    sentieon_licence_dir: Path,
    sentieon_licence_server: str,
```

Reply:
Yeah I don't see why not 🤔 I can't think of a use-case for it at the moment but it could be useful, and it feels good to set these Sentieon variables in a consistent way